bin/post_compile

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+indent() {
+  sed "s/^/       /"
+}
+
+echo "-----> Running migrations..."
```

Comment:
You might be able to use the existing `puts-step`: https://github.com/heroku/heroku-buildpack-python/blob/6f6b75bb1f129044670fb0ab43680af762f66739/bin/utils#L20